unified_range/utils.py

```diff
@@ -47,15 +47,19 @@ def is_unified_range(rng):
     return _is_unified_ops(rng) and not _is_semver_ops(rng)
 
 
-def transform_to_semver(unified_spec: str) -> str:
+def transform_to_semver(unified_spec: str, comma_separated: bool) -> str:
```

Comment:
I opted for a no default required argument since this is not part of the (intended) api, and making callers aware of the option might be beneficial. Setting a default `False` value is also a viable option imo.

Reply:
Did you validate that all the places we had currently are with `comma_separated=False`?

Reply:
yes.